### DISCUSSION TITLE
aff_log not updating

### DISCUSSION THREAD

**Discussion by RepressedNerd** [Monika-After-Story/MonikaModDev#7939] [QA - tech support + general] [answered]
Hello!

I just realised that the aff_log hasn't been updated since I moved from Windows to Mac. I read that there used to be issues with Python on Macs regarding logs, but I've also read that the issue was fixed. 

So, I was wondering, is it different because I switched OS? Is there a new aff_log file hidden? And what happened to all the affection I would have gained since then? Did I never gain any affection?

Thanks in advance!

**Comment by KritRom** [comment]
Are you checking your Affection through Terminal or through the file in the `log` folder?

**Comment by ghost** [comment, marked as answer]
in the log folder, for mac, there will be nothing in there (im on macbook too). here is how to check it on mac.


https://github.com/Monika-After-Story/MonikaModDev/wiki/All-Features-%5BSPOILERS%5D#how-to-view-your-affection
